app/components/publications/publication/translations/translations-info-panel.hbs

```diff
@@ -0,0 +1,96 @@
+<Auk::Panel class={{if this.isEditing "auk-panel--editing"}}>
+  <Auk::Panel::Header>
+    <Auk::Toolbar>
+      <Auk::Toolbar::Group @position="left">
+        <Auk::Toolbar::Item>
+          <h4 class="auk-panel__title">
+            {{t "translation-info"}}
+          </h4>
+        </Auk::Toolbar::Item>
+      </Auk::Toolbar::Group>
+      <Auk::Toolbar::Group @position="right">
+        <Auk::Toolbar::Item>
+          <Auk::Button
+            @skin="tertiary"
+            @icon="pencil"
+            @disabled={{not (or this.publicationStatus.isPending this.publicationStatus.isPaused)}}
+            @loading={{this.loadStatus.isRunning}}
+            @layout="icon-only"
+            {{on "click" this.openEditingPanel}}
+          />
+        </Auk::Toolbar::Item>
+      </Auk::Toolbar::Group>
+    </Auk::Toolbar>
+  </Auk::Panel::Header>
+
+  {{#if this.isEditing}}
+    <Auk::Panel::Body>
+      <div class="auk-o-grid">
+        <div class="auk-o-grid-col-3">
+          <div class="auk-form-group">
+            <Auk::Label>{{t "translation-due-date"}}</Auk::Label>
+            <Auk::Datepicker
+              @date={{this.dueDate}}
+              @onChange={{this.setTranslationDueDate}}
+            />
+            {{#if
+                (is-after this.dueDate precision="days")
+            }}
+              <Auk::FormHelpText
+                @text={{t "date-expired"}}
+                @skin="warning"
+                @icon="alert-triangle"
+              />
+            {{/if}}
```

Comment:
Bij de drukproeven is er hier een extra check om de overdue warning niet te tonen voor afgeronde publicaties.